packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -15,18 +15,15 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
       borderColor: 'transparent',
     },
     outline: {
-      ...getOutlineColorProps({ color: t.colors.brandForeground1, colorDark: t.colors.brandForeground1 }, t),
+      ...getOutlineColorProps({ color: t.colors.brandForeground1 }, t),
```

Comment:
It looks like the default is the same as brand, can we remove a duplicate somehow?

Reply:
Yes. I was thinking where its better to have default values. Because lookup function uses "brand filled" as default. In this case we don't need to add anything to default tokens. I'll remove defaults